From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@rcarmo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rucarmo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@rcarmo : Thanks for your contribution to the Azure documentation! The author, @davidmu1, has been notified to review your proposed change.

squillace commented:
@rcarmo this is a good fix! Sorry about that. :-) Don't know WHAT  we were thinking. ;-) @davidmu1 LGTM....FWIW

rcarmo commented:
@squillace nah, no worries. I know it's hard to do docs -- you stare at them for so long sometimes that things like this get by.

davidmu1 commented:
@rcarmo: Thank you very much for your feedback on the article.

davidmu1 commented:
#sign-off

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Resolving merge conflict

v-alje committed:
Merge branch 'release-mvc-b2c' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-b2c

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Resolving merge conflict

davidmu1 committed:
mvc work

davidmu1 committed:
warning corrections

v-alje committed:
Bringing even with master.

davidmu1 committed:
minor corrections

davidmu1 committed:
image test

davidmu1 committed:
image test 2

davidmu1 committed:
minor corrections

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

carolinacmoravia committed:
C14101: Inconsistent format on target version due to missing breaklines

Hello, @ponatara, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

v-alje committed:
Bringing even with master.

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

davidmu1 committed:
incorp feedback

v-alje committed:
Bringing even with master.

davidmu1 committed:
eng feedback

davidmu1 committed:
eng feedback

v-alje committed:
Bringing even with master.

davidmu1 committed:
incorp feedback

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

dlepow committed:
Batch: update storage guidance

dlepow committed:
typo

dlepow committed:
typos

davidmu1 committed:
landing correction

davidmu1 committed:
Merge branch 'release-mvc-b2c' of https://github.com/Microsoft/azure-docs-pr into b2cmvc3

dlepow committed:
Link fix

davidmu1 committed:
landing correction

dlepow committed:
link fix

v-alje committed:
Bringing even with master.

davidmu1 committed:
incorp feedback

davidmu1 committed:
coorected text on line 73 of  index.yml

v-alje committed:
Bringing even with master.

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

fhryo-msft committed:
add "Azure Storage metrics migration"

add "Azure Storage metrics migration"

fhryo-msft committed:
update based on grammar check

update based on grammar check

fhryo-msft committed:
update based on grammar check

update based on grammar check

fhryo-msft committed:
Revise based on review

Revise based on review

v-alje committed:
Bringing even with master.

GitHubber17 committed:
Merge pull request #36875 from davidmu1/b2cmvc3

mvc work

fhryo-msft committed:
revise based on review

revise based on review

roygara committed:
TOC changes.

fhryo-msft committed:
article updates.

fhryo-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

roygara committed:
Fixing TOC urlss

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into TOCMetrics

fhryo-msft committed:
Merge pull request #1 from roygara/TOCMetrics

TOC changes.

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

fhryo-msft committed:
revise based on review

revise based on review

fhryo-msft committed:
Merge branch 'master' of https://github.com/fhryo-msft/azure-docs-pr

yoelhor committed:
Update active-directory-b2c-setup-commonaad-custom.md

Minimized the instructions, to make the article easy to read and using same format as the rest of the IDP articles.

1. Minimized the prerequisites section
2.  Steps number
3. App secret name
4. Change the ValidTokenIssuerPrefixes to https://sts.windows.net/
5. Minimized the claims provider and technical configuration. Since we already provided the XML
6. Copy user journey as optional
7. Uploading the testing the policy as the rest of B2C articales

carolinacmoravia committed:
C15815: LOCBLOCKER: Missing ``` causing content blocked for localization

Hello, @ckarst,

Localization team has reported source content issue that causes localizable text to be blocked from localization.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

fhryo-msft committed:
Fix doc issues

fhryo-msft committed:
revise based on check

fhryo-msft committed:
revise based review

divyaswarnkar committed:
EHupdate1

Commit:
pm updates

v-alje committed:
Bringing even with master.

divyaswarnkar committed:
EHupdate2

divyaswarnkar committed:
EHupdate3

divyaswarnkar committed:
EHupdate4

divyaswarnkar committed:
EHupdate5

v-alje committed:
Bringing even with master.

TerryLanfear committed:
date update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

divyaswarnkar committed:
EHupdate7

divyaswarnkar committed:
EHupdate7

fhryo-msft committed:
Revise to reflect latest dimension

Revise to reflect latest dimension

v-alje committed:
Bringing even with master.

Commit:
Task 1090: Transform article into tutorial

fhryo-msft committed:
Update latest effective dimension value

Update latest effective dimension value

bwren committed:
updates

genlin committed:
break the articles

Commit:
automation : md5 hash addition for inventory

TerryLanfear committed:
date change

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

genlin committed:
correct block issue

genlin committed:
correct a typo

genlin committed:
correct broken link issue

Commit:
add preview indication

Commit:
fix error

Commit:
BatchAI. Update quickstart-python.md

Updated the doc to match the latest API.
Note, the new instructions still work with older API.

Commit:
fix errors again

Commit:
Addressing CR issues

v-kents committed:
checking indent

Commit:
Update quickstart-python.md

More fixes

Commit:
Update quickstart-python.md

Fixed a misprint

mimig1 committed:
Cosmos DB: Updating quickstart to use VS Code

Commit:
Update quickstart-python.md

Removed an extra space

Commit:
Re-factored templates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-templates

v-kents committed:
add section title

vasivara-MSFT committed:
(ForumsTeam) fixes MicrosoftDocs/azure-docs#7126

When creating table @a, column date is missing

Commit:
Template re-factors

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-templates

v-kents committed:
Add item type

v-alje committed:
Bringing even with master.

mimig1 committed:
more updates

Commit:
Fix.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-templates

v-kents committed:
change html to text

v-kents committed:
surround text/html with quotes

mimig1 committed:
Removed random New Folder

v-kents committed:
convert to paragraph block

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-1090

Commit:
Task 1090: Add screenshot, metadate to include files

v-kents committed:
add title back

v-kents committed:
test removing html

v-kents committed:
remove notice section

Commit:
fixes for github issue 7162

v-kents committed:
test adding to abstract

mimig1 committed:
Corrected typo.

Commit:
removed a word

ganganarayanan committed:
Updates to article

Updates to article

mimig1 committed:
Merge branch 'master' into 04132018python

ganganarayanan committed:
Updated spelling

Updated spelling

ganganarayanan committed:
Added new line

Added new line

daveba committed:
updating and adding tutorials

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into add-tutorials-update-toc

daveba committed:
updates

bwren committed:
updates

mimig1 committed:
Moved interpreter info into body of QS

mimig1 committed:
Merge branch '04132018python' of https://github.com/mimig1/azure-docs-pr into 04132018python

daveba committed:
updates

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into add-tutorials-update-toc

divyaswarnkar committed:
EHupdate8

douglaslMS committed:
Indenting fixes to prevent rendering as code blocks.

MicrosoftGuyJFlo committed:
[Azure AD] SSPR - PHS not requirement for Password Writeback

bwren committed:
fix

SnehaGunda committed:
Deleting image files

PRMerger17 committed:
Merge pull request #37838 from MicrosoftGuyJFlo/379687

[Azure AD] SSPR - PHS not requirement for Password Writeback

bwren committed:
acrolinx

ktoliver committed:
Merge pull request #37782 from v-savila/1235111

add preview indication

MicrosoftGuyJFlo committed:
[MFA] Trusted IPs - 389860 - GitHub Issue #5826

SnehaGunda committed:
Deleting images

bwren committed:
acrolinx

barbkess committed:
remove 3 integration files

bwren committed:
Merge branch 'master' into kusto-ga-1

barbkess committed:
fix power bi link

ktoliver committed:
Merge pull request #37735 from dksimpson/dean-1090

Task 1090: Transform article into tutorial

TerryLanfear committed:
fix to images

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-mig

daveba committed:
renamed file to mee quality review

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into add-tutorials-update-toc

TerryLanfear committed:
update to images

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#7161

CawaMS committed:
change asp.net core telemetry sample code

daveba committed:
renamed file to mee quality review

PRMerger16 committed:
Merge pull request #37844 from MicrosoftGuyJFlo/389860

[MFA] Trusted IPs - 389860 - GitHub Issue #5826

Ja-Dunn committed:
Merge pull request #37828 from ganganarayanan/master

Updates to article

ktoliver committed:
Merge pull request #37833 from daveba/add-tutorials-update-toc

[Azure AD MSI] Add tutorials update toc

barbkess committed:
fix redirects

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into topic-updates5

CawaMS committed:
fix typo

zhiweiwangmsft committed:
Update active-directory-aadconnect-health-alert-catalog.md

v-alje committed:
Bringing even with master.

zhiweiwangmsft committed:
update minor

genlin committed:
delete metadata

Commit:
updates

douglaslMS committed:
Added one more curly brace to JSON snippet.

PRMerger10 committed:
Merge pull request #37835 from douglaslMS/adf-fix-indents

Indenting fixes to prevent rendering as code blocks.

PRMerger15 committed:
Merge pull request #37797 from vasivara-MSFT/patch-3

(ForumsTeam) fixes MicrosoftDocs/azure-docs#7126

PRMerger10 committed:
Merge pull request #37823 from JimacoMS2/sf-linux-get-started-issue-7162

fixes for github issue 7162

KumudD committed:
removed preview text

PRMerger14 committed:
Merge pull request #37860 from KumudD/haportsupdate0413

removed preview text

barbkess committed:
freshness and metadata updates

barbkess committed:
freshness and metadata updates

barbkess committed:
fix acrolinx

barbkess committed:
Freshness and metadata updates

barbkess committed:
acrolinx fixes

giladmit committed:
added Additional Information section

Commit:
Merge pull request #37877 from giladmit/GiladWorkingBranch

added Additional Information section

PRMerger14 committed:
Merge pull request #37855 from zhiweiwangmsft/patch-33

Update ADFS 2016 remediation

TerryLanfear committed:
changed date

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aac-asc

barbkess committed:
fix getdate and timestamp

GitHubber17 committed:
Merge pull request #35391 from bwren/kusto-ga-1

Kusto GA - Alert actions

GitHubber17 committed:
Merge pull request #37435 from fhryo-msft/master

add "Azure Storage metrics migration"

GitHubber17 committed:
Merge pull request #37630 from divyaswarnkar/eventhub

Eventhub

GitHubber17 committed:
Merge pull request #37769 from genlin/990

allocation-failure-classic.md

GitHubber17 committed:
Merge pull request #37841 from SnehaGunda/ASAOverview

Deleting image files

GitHubber17 committed:
Merge pull request #37845 from barbkess/topic-updates5

remove 3 integration files

GitHubber17 committed:
Merge pull request #37851 from TerryLanfear/sept-mig

Sept mig

GitHubber17 committed:
Merge pull request #37863 from barbkess/topic-updates6

freshness and metadata updates

GitHubber17 committed:
Merge pull request #37868 from barbkess/topic-updates7

freshness and metadata updates

GitHubber17 committed:
Merge pull request #37874 from barbkess/topic-updates8

Freshness and metadata updates

parakhj committed:
Merge pull request #1 from yoelhor/patch-1

Update active-directory-b2c-setup-commonaad-custom.md

parakhj committed:
Minor changes

parakhj committed:
Minor changes

parakhj committed:
Minor edits

barbkess committed:
articles fixes

barbkess committed:
freshness review and metadata updates

Sogup committed:
Update backup-azure-arm-vms-prepare.md

PRMerger19 committed:
Merge pull request #37895 from Sogup/patch-52

Update backup-azure-arm-vms-prepare.md

Sogup committed:
Update backup-azure-vms-introduction.md

PRMerger13 committed:
Merge pull request #37896 from Sogup/patch-55

Update backup-azure-vms-introduction.md

cephalin committed:
update db firewall command

PRMerger14 committed:
Merge pull request #37907 from cephalin/outbound

update db firewall command

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-templates

Commit:
Added metadata.

v-geberr committed:
timeout faq

PRMerger15 committed:
Merge pull request #37561 from carolinacmoravia/patch-675

C15815: LOCBLOCKER: Missing ``` causing content blocked for localization

PRMerger19 committed:
Merge pull request #37917 from v-geberr/0416-timeout-login

timeout faq

Commit:
updated date

itechedit committed:
Merge pull request #37430 from TerryLanfear/aac-asc

Aac asc

Commit:
Merge pull request #36995 from carolinacmoravia/patch-659

C14101: Inconsistent format on target version due to missing breaklines

mrbullwinkle committed:
add az320820.vo.msecnd.net

rjagiewich committed:
Merge pull request #37801 from jimdial/vnet-templates

VNet template samples

itechedit committed:
Merge pull request #37586 from MicrosoftDocs/release-mvc-b2c

Release mvc b2c

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

dlepow committed:
Added metadata to include

itechedit committed:
Update active-directory-b2c-setup-commonaad-custom.md

PRMerger19 committed:
Merge pull request #37920 from mrbullwinkle/mrb_04_16_2018_ipaddress

add az320820.vo.msecnd.net

Commit:
autohashalgo

itechedit committed:
Merge pull request #37882 from parakhj/commonaad

A round of updates

itechedit committed:
Merge pull request #37884 from barbkess/master

articles fixes

rjagiewich committed:
Merge pull request #37885 from barbkess/topic-updates10

freshness review and metadata updates

itechedit committed:
Merge pull request #37794 from mimig1/04132018python

Cosmos DB: Updating quickstart to use VS Code

Commit:
Merge pull request #37775 from georgewallace/autohashalgo

automation : md5 hash addition for inventory

PRMerger16 committed:
Merge pull request #37852 from CawaMS/master

change asp.net core telemetry sample code

itechedit committed:
Merge pull request #37786 from AlexanderYukhanov/master

BatchAI. Update quickstart-python.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7213

Typos on the NIC count/network bandwidth for the sizes Standard_D3_v2 and Standard_D4_v2

itechedit committed:
Merge pull request #37289 from dlepow/fix

Batch: updates to storage guidance

rjagiewich committed:
Merge pull request #37919 from davidmu1/pmupdate1

Pmupdate1

douglaslMS committed:
Added line to Data Factory Contributor role table per PM.

v-alje committed:
Merge pull request #37926 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #37929 from douglaslMS/adf-add-role

Added line to Data Factory Contributor role table per PM.

Commit:
Merge pull request #37927 from MicahMcKittrick-MSFT/patch-18

Fixes MicrosoftDocs/azure-docs#7213

Commit:
Merge pull request #37850 from tfitzmac/0413resources

fixes MicrosoftDocs/azure-docs#7161
